src/resources/js/validation.js

```diff
@@ -217,6 +497,29 @@ tribe.validation = {};
 	};
 
 	/**
+	 * Validates a single Field
+	 *
+	 * @since  TBD
+	 *
+	 * @param  {string}  message  Message to be Attached
+	 * @param  {object}  $field   jQuery Object for the field
+	 * @param  {object}  $list    jQuery Object for list of Errors
+	 *
+	 * @return {void}
+	 */
+	obj.addErrorLine = function( message, $field, $list ) {
+		var $listItem = $( '<li>' ).text( message );
+
+		// Add which field has thrown the error
+		$listItem.data( 'validationField', $field );
+
+		// Add which notice item is related to this error field
+		$field.data( 'validationNoticeItem', $field );
+
+		$list.append( $listItem );
+	};
+
+	/**
 	 * Hooks to the submit and if invalid prevents submit to happen
```

Comment:
"prevents submit from completing"? Same below (538) "prevent the submit from completing"